package.json

```diff
@@ -53,8 +53,9 @@
     "@babel/preset-typescript": "^7.26.0",
     "@chromatic-com/storybook": "^3",
     "@csstools/postcss-sass": "^5.1.1",
-    "@cypress/browserify-preprocessor": "^3.0.2",
     "@cypress/code-coverage": "^3.13.9",
+    "@eslint/eslintrc": "^3.2.0",
+    "@eslint/js": "^9.17.0",
```

Comment:
I suggest you [add a group for `eslint` dependencies in `dependabot.yml`](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/e2485418b0035cc0d2b8739a46465036e9109399/.github/dependabot.yml#L20-L20).

It seems like we now have multiple packages within the `@eslint` namespace. It is likely that we want to update these together like we do for storybook and others.